package.json

```diff
@@ -19,6 +19,7 @@
     "hooks:install": "node node_modules/husky/husky.js install",
     "hooks:uninstall": "node node_modules/husky/husky.js uninstall",
     "lint": "eslint .",
+    "lint:curriculum": "markdownlint -r markdown-yaml.js -c .markdownlintrc.json curriculum/challenges/english/**/*.md",
```

Comment:
Throw this in a script? https://github.com/freeCodeCamp/freeCodeCamp/tree/master/tools/scripts/ci